### ISSUE TITLE
Limited linking to special pages like 'genindex' 

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#46]
Generated pages like //genindex// can be referenced with alternative name:

```
#!restructuredtext

Works:
    See :ref:`genindex`

Does not work:
    See :ref:`index listing <genindex>`

How to make a reference like?
    See :ref:<genindex#A>`
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/46
- Originally reported by: Anonymous
- Originally created at: 2008-11-18T17:43:13.500

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2008-11-22 12:24:00+00:00_

I'll try to find a way to do this.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2008-12-28 23:00:01+00:00_

Changeset <<cset 1aa8baa0d82520c263f72fac4b156fadc97da22b>>:

Fix #46: allow linking to special labels via :ref:`abc <genindex>`.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-08-25 12:19:54+00:00_

Removing milestone: 0.6 (automated comment)
